build.yaml

```diff
@@ -7,6 +7,7 @@ settings:
   '#3': Use "-preN" suffixes to identify pre-release versions
   '#4': Per-language overrides are possible with (eg) ruby_version tag here
   '#5': See the expand_version.py for all the quirks here
+  g_stands_for: good
```

Comment:
To be clear: let's have 'good' here means morally good.

And that avoids someone saying, "Make gRPC great again." It's always been great.